app/helpers/plagiarism_checker_helper.rb

```diff
@@ -0,0 +1,115 @@
+module PlagiarismCheckerHelper
+  require 'simicheck_webservice'
+  require 'submission_content_fetcher'
+
+  # PlagiarismCheckerHelper acts as the integration point between all services and models
```

Comment:
We discussed this briefly in the meeting today, but just to document here with some more points.  Reasoning for having as a helper is to 
- attempt at paralleling existing code in Expertiza,
- deal with the fact that there isn't a request path to handle (rather a delayed task), and
- allow for a Simicheck comparison to be started from anywhere which makes code changes later easier
Also when the Simicheck server finishes the comparison it notifies the Expertiza server with a request to a callback URL that is provided, and that is handled in PlagiarismCheckerComparisonController.  It seems like how we wrote the controller demonstrates the last point above.  Does this resolve your question?